mailbox/backup/src/main/java/org/apache/james/mailbox/backup/MailAccountZipIterator.java

```diff
@@ -0,0 +1,47 @@
+package org.apache.james.mailbox.backup;
```

Comment:
Careful to licenses!

Reply:
Thanks i have forgotten to configure intellij to include them automatically on my new computer.
Will do it ASAP